resources/js/Pages/Layout.vue

```diff
@@ -114,12 +115,25 @@ export default defineComponent({
                 document.removeEventListener('touchstart', handleOutsideClick);
             },
         },
+        resize: {
```

Comment:
What is the advantage of a directive here? 
Would it be simpler to use [ResizeObserver](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver)? 

See example on [StackOverflow](https://stackoverflow.com/a/60888674) (will need to be slightly modified to fit our usecase).

Reply:
I implemented a directive because the v-resize doesn't exist. https://vuejs.org/guide/reusability/custom-directives.html#introduction , to me a directive seems like a more natural solution. but now that you mention it... this is not only going to be fired on resize, it will also be fired on click so yeah, so a directive there is not the most appropriate use case. :thinking:

Reply:
Done.